ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -744,9 +741,8 @@ generateGhcLibParserCabal ghcFlavor = do
         , "    build-depends:"
         ] ++ indent2 (withCommas (commonBuildDepends ghcFlavor)) ++
         [ "    build-tools: alex >= 3.1, happy >= 1.19.4"
-        , "    other-extensions:"
-        ] ++
-        indent2 (askField lib "other-extensions:") ++
+        , "    other-extensions:" ] ++ indent2 (askField lib "other-extensions:") ++
+        [ "    default-extensions:" ] ++ indent2 (askField lib "default-extensions:") ++
```

Comment:
This looks like we have the same code twice. Is there a specific reason for this?

Reply:
Two functions for two cabal files : `ghc-lib-parser.cabal` and `ghc-lib.cabal`. No deeper reason than that.

Reply:
Unless I misunderstand your question and you are confusing lines 744 and 745 as being the same? (They're not they differ in field-name).

Reply:
Your first answer answers my question. Thanks a lot.